bookwyrm/templates/directory/user_card.html

```diff
@@ -20,7 +20,7 @@
 
         <div>
         {% if user.summary %}
-            {{ user.summary|to_markdown|safe|truncatechars_html:40 }}
+            {{ user.summary|to_markdown|safe|truncatechars_html:81 }}
```

Comment:
I had to increase the truncatechars_html allowance, since on the profile cards that would display in the user search, the truncatechars_html was counting about 41 characters of the CSS bloc (despite it not being visible). This caused it to cut off the entire profile summary preview. Increasing the cap to 81 preserves the original preview limit of 40.